javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj){
+  let returnArray = [];
+
+  for (const property in obj) {
+    returnArray.push(`<li>${property}: ${obj.name}</li>`);
+    returnArray.push(`<li>${property}: ${obj.age}</li>`);
+  }
+  returnArray.splice(1,1);
+  returnArray.splice(1,1);
+  //console.log(returnArray);
+  return returnArray;
+}; // Okay I know this probably wasn't how it was supposed to be solved, but it works ;v
```

Comment:
Good job on reaching a solution! :) 👍 

If you want to try and simplify this a tad/practice with more methods here is another approach:

- We can think of which methods will allow us to work with objects and that gives us an arr (some are [Object.entries()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries), [Object.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys), map())
- Perhaps we can use Object.entries or keys, passing in the `obj`. From there, we can map over every element and return the array of the new key value pairs using syntax similar to what you have inside the push() on line 25.